buildscripts/travis-build.sh

```diff
@@ -55,6 +55,7 @@ printf "\n"
 
 
 echo "Running : make kubegen"
+make kubegendelete
```

Comment:
this may not be required if above comment is taken care of

Reply:
done